dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -939,6 +939,24 @@ def test_min_col_samples_used(self, *mocks):
                                                samples_per_update=1)
         self.assertEqual(2, sparse_profile._min_col_samples_used)
 
+    @mock.patch('dataprofiler.profilers.profile_builder.DataLabeler')
```

Comment:
can also mock out update, no need to profile anything in this.

Reply:
Probably should test for Unstructured and dp.Profiler too, no?

Reply:
update not mocked since I wanted to make sure None still worked without error (it gets manipulated in update). Also the test takes 30ms so its not really going to get much faster. 

I added the tests for unstructured and dp.Profiler too